### PR TITLE
fix(guide): dont-block-the-event-loop.md

### PR DESCRIPTION
**Pull request by objectisundefined**
incorrect example code

### CODE REVIEW

**Comment by XhmikosR** [comment]
This will need to be applied to translations too, if it's approved.

**Comment by objectisundefined** [comment]
@XhmikosR ok

File: locale/zh-cn/docs/guides/dont-block-the-event-loop.md [hunk 1]
```diff
@@ -256,19 +256,19 @@ app.get('/redos-me', (req, res) => {
 var obj = { a: 1 };
 var niter = 20;
 
-var before, res, took;
+var before, str, pos, res, took;
 
 for (var i = 0; i < niter; i++) {
-  obj = { obj1: obj, obj2: obj }; // 每个循环里面将对象 size 加倍
+  obj = { obj1: obj, obj2: obj }; // Doubles in size each iter
```
Comment (XhmikosR) [inline locale/zh-cn/docs/guides/dont-block-the-event-loop.md]: The comments could stay since they are translated.

Comment (objectisundefined) [reply locale/zh-cn/docs/guides/dont-block-the-event-loop.md]: Fixed.

**Comment by Trott** [comment]
It's not clear to me that the existing example code is incorrect. This seems to use three different variables for no reason. (Or maybe it's more clear that way? Probably makes sense to rename `res` if that's the case?)

If you don't like the variable re-use, another option is to get rid of the variable assignments entirely. The point of the code is the impact of calling `JSON` methods. The variable assignments are irrelevant, although I guess they make the code seem less contrived.

**Comment by Trott** [comment]
(My comments above are non-blocking. If everyone else thinks this is an improvement, then that's fine by me. I don't think this change makes it any worse or anything like that.)

**Comment by objectisundefined** [comment]
@Trott variable `str` in the existing example code was not declared.

**Comment by Trott** [comment]
> @Trott variable `str` in the existing example code was not declared.

Ah! I missed that! Thanks! Yes, LGTM!

**Review by Trott — APPROVED** [review]

**Review by ghost — APPROVED** [review]
Good catch!

**Pull request merged**